core/src/main/java/io/snappydata/impl/SnappyHiveCatalog.java

```diff
@@ -328,7 +328,7 @@ public Object call() throws Exception {
           Object value = parameters.get(StoreUtils.GEM_INDEXED_TABLE());
           String baseTable = value != null ? value.toString() : "";
           String dmls = JdbcExtendedUtils.
-              getInsertOrPutString(fullyQualifiedName, schema, false);
+              getInsertOrPutString(fullyQualifiedName, schema, false, false);
```

Comment:
Normally string created will have quotes which works just fine for prepareStatement etc. The case where this has to be escaped is when the generated string is embedded into generated code i.e. a string like 'prepareStatement("SELECT ..."COL1"...")' which is compiled. Here the quotes have to be escaped.